client.go

```diff
@@ -412,6 +414,14 @@ func (c *Client) SetHeader(name, value string) {
 	c.resty.SetHeader(name, value)
 }
 
+func (c *Client) SanitizeAuthorizationHeader() {
```

Comment:
Would it make sense to make this function private since it's only called from Client.SetDebug(...)?

Reply:
I have now made this a private function